examples/core/gated_gcn/train.py

```diff
@@ -98,12 +100,37 @@ def evaluate(model, device, data_loader, evaluator):
     return evaluator.eval({"y_true": y_true, "y_pred": y_pred})["rocauc"]
 
 
-def main():
-    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        default="cora",
+        help="Dataset name ('cora', 'citeseer', 'pubmed').",
+    )
+    parser.add_argument(
+        "--num_epochs",
+        type=int,
+        default=200,
+        help="Number of epochs for train.",
+    )
+    parser.add_argument(
+        "--num_gpus",
+        type=int,
+        default=0,
+        help="Number of GPUs used for train and evaluation.",
+    )
+    args = parser.parse_args()
+    print(f"Training with DGL built-in GATConv module.")
 
     # Load ogb dataset & evaluator.
-    dataset = DglGraphPropPredDataset(name="ogbg-molhiv")
-    evaluator = Evaluator(name="ogbg-molhiv")
+    dataset = DglGraphPropPredDataset(name=args.dataset)
```

Comment:
are you sure datasets like `cora`, `pubmed` you specified in `args.dataset` supported by `DglGraphxxx` which is for graph property prediction for ogb datasets? Please make sure it really work.

Reply:
`GraphDataLoader` is instantiated.

Reply:
> are you sure datasets like `cora`, `pubmed` you specified in `args.dataset` supported by `DglGraphxxx` which is for graph property prediction for ogb datasets? Please make sure it really work.

Yes, I have tested locally. This API would download dataset and then load it.

Reply:
use ogbn dataset here.